### ISSUE TITLE
Cannot read property 'email' of undefined

### ISSUE THREAD

**Issue by MrVu** [EddyVerbruggen/nativescript-plugin-firebase#118]
I'm trying to make a register and login page with firebase and run into this error "Error in firebase.createUser: TypeError: Cannot read property 'email' of undefined". This is my code. Is anyone had this issue before please help me. Thank you
![screenshot from 2016-08-30 12 20 37](https://cloud.githubusercontent.com/assets/17061628/18074559/a93ef146-6eac-11e6-8804-be1b4e657c04.png)

![screenshot from 2016-08-30 12 26 34](https://cloud.githubusercontent.com/assets/17061628/18074616/0d35cc9c-6ead-11e6-991c-30cffff40942.png)

**Comment by EddyVerbruggen** [comment]
Which platform?

**Comment by MrVu** [comment]
I'm using android

**Comment by EddyVerbruggen** [comment]
Can you check `adb logcat` and paste the error here? I'd like to know which line of code in which file has an `undefined` object `email` is invoked upon.

**Comment by MrVu** [comment]
This is the logcat I got. I think the problem come from here
08-30 20:41:03.461  1648  1661 W GooglePlayServicesUtil: Google Play Store is missing.
08-30 20:41:06.499  1648  1661 W DynamiteModule: Local module descriptor class for com.google.firebase.auth not found

I installed google play service and Google Repository.Am I missing something?
![screenshot from 2016-08-31 10 13 11](https://cloud.githubusercontent.com/assets/17061628/18112022/975efa72-6f63-11e6-8a82-9f97d4ced62e.png)
